eclass/python-utils-r1.eclass

```diff
@@ -1038,12 +1048,17 @@ python_fix_shebang() {
 			local shebang i
 			local error= from=
 
-			IFS= read -r shebang <"${f}"
+			# normally ${f} starts with ${path}
+			if [[ ${f} != "${path}"* && ${f} -ne 0 ]]; then
+				die "'find ${path} -type f -print0' failed"
+			fi
```

Comment:
Please take a look at this for solution to subshell dying. Do you like it or not? ;-P